xcp_d/utils/restingstate.py

```diff
@@ -178,6 +184,8 @@ def compute_alff(data_matrix, low_pass, high_pass, TR, sample_mask=None):
         # from the value closest to the low pass cutoff, to the value closest
         # to the high pass pass cutoff
         alff[i_voxel] = len(ff_alff) * np.mean(power_spectrum_sqrt[ff_alff[0] : ff_alff[1]])
+        # Rescale ALFF based on original BOLD scale
+        alff[i_voxel] *= sd_scale
```

Comment:
ohh so both periodogram and lombscargle get the scaling and rescaling?

Reply:
Yes, though it's probably unnecessary for the periodogram version.